### PR TITLE
WIP upgrade cassandra docker image to 3.11

### PR DESCRIPTION
**Pull request by mbaechler**
check support ... and speed

### CODE REVIEW

**Comment by chibenwa** [comment]
Would you plan to upgrade prod if it is an enhancement?

File: backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ public DockerCassandraRule() {
             new ImageFromDockerfile("cassandra_2_2_10", deleteOnExit)
```
Comment (rouazana) [inline backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java]: name :)

**Comment by chibenwa** [comment]
Can't the failing test you commented be relayed to the prod issue?

Do this looks like "missing mbx syndrom"??

If yes, could you :
 - try to solve it by turning row cache off?
 - using a real composite primary key instead of UDT?

Just curious ;-)

**Comment by mbaechler** [comment]
test this please

**Comment by mbaechler** [comment]
test this please

**Comment by mbaechler** [comment]
test this please

**Comment by mbaechler** [comment]
test this please

**Comment by mbaechler** [comment]
replaced by https://github.com/apache/james-project/pull/77

**Pull request closed without merging**